Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -83,7 +83,7 @@ extension BrowserViewController {
     
     struct MenuTabDetailsView: View {
         @SwiftUI.Environment(\.colorScheme) var colorScheme: ColorScheme
-        var tab: Tab?
+        weak var tab: Tab?
```

Comment:
While this technically fixes the tab issue, we should fix the underlying leak in the menu as well